packages/aws-cdk/lib/cli-arguments.ts

```diff
@@ -0,0 +1,1253 @@
+// -------------------------------------------------------------------------------------------
+// GENERATED FROM packages/aws-cdk/lib/config.ts.
+// Do not edit by hand; all changes will be overwritten at build time from the config file.
+// -------------------------------------------------------------------------------------------
+/* eslint-disable @typescript-eslint/comma-dangle, comma-spacing, max-len, quotes, quote-props */
+/**
+ * The structure of the CLI configuration, generated from packages/aws-cdk/lib/config.ts
+ *
+ * @struct
+ */
+export interface CliArguments {
+  /**
+   * The CLI command name followed by any properties of the command
+   */
+  readonly _: Array<string>;
+
+  /**
+   * Global options available to all CLI commands
+   */
+  readonly globalOptions?: GlobalOptions;
+
+  /**
+   * Lists all stacks in the app
+   *
+   * aliases: ls
+   */
+  readonly list?: ListOptions;
+
+  /**
+   * Synthesizes and prints the CloudFormation template for this stack
+   *
+   * aliases: synth
```

Comment:
I don't think we should care too much about the aliases, but I also don't see any harm in including the information, especially if we end up committing this file.